content/docs/started/workstation/minikube-linux.md

```diff
@@ -0,0 +1,237 @@
+
+The following topics will be covered:
+
+- installation of docker-community edition (docker-ce), kubectl, and minikube
+- installation of Kubeflow
+- launch of Kubeflow central dashboard
+- compilation of a pipeline example
+- upload of compilated application to Pipeline UI
+- execution of the uploaded application
+
+## Prerequistes
+
+ - Ubuntu 18 machine with min 8 cores, 16GB RAM, 250GB storage
+ - Root privilege on a ubuntu machine
+
+
+## Installation of docker-ce, kubectl, and minikube
+Minikube provides a no-driver mode based on docker without requiring a hypervisor.
+
+### Install docker-ce
+```
+apt-get update
+apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+apt-get update
+apt-get install docker-ce docker-ce-cli containerd.io
+```
+To verify your installation, run 
+```
+docker run hello-world
+```
+The expected output should like this:
+```
+Unable to find image 'hello-world:latest' locally
+latest: Pulling from library/hello-world
+1b930d010525: Pull complete                                                                                        Digest: sha256:fc6a51919cfeb2e6763f62b6d9e8815acbf7cd2e476ea353743570610737b752
+Status: Downloaded newer image for hello-world:latest
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+```
+
+### Install kubectl
+`kubectl` is a Kubernetes command-line tool that allows you to run commands against Kubernetes clusters. Following instruction will install the version 1.15 of `kubectl`, if you are looking for a specific version, see [official instruction](https://kubernetes.io/docs/tasks/tools/install-kubectl/). 
+```
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.15.0/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+mv ./kubectl /usr/local/bin/kubectl
+```
+
+
+### Install minikube
+
+Following instruction will install minikube v1.2.0, if you are looking for a specific version, see [minikube official site](https://github.com/kubernetes/minikube/releases).
+```
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.2.0/minikube-linux-amd64
+```
+Move to /usr/local/bin directory
+```
+chmod +x minikube
+cp minikube /usr/local/bin/
+rm minikube
+```
+
+### Start minikube
+Following command will start minikube with 6 CPUs, 12288 memory, 120G disk size. 
+```
+minikube start --vm-driver=none --cpus 6 --memory 12288 --disk-size=120g --extra-config=apiserver.authorization-mode=RBAC --extra-config=kubelet.resolv-conf=/run/systemd/resolve/resolv.conf --extra-config kubeadm.ignore-preflight-errors=SystemVerification
+```
+
+## Installation of Kubeflow
+
+The following instruction is for installing Kubeflow v1.0 under /root/kubeflow/v1.0 directory. 
+```
+mkdir -p /root/kubeflow/v1.0
```

Comment:
Do you think it's necessary to include the version number in the path, or should we generalize the path for simplicity? (So that we don't have to update the path when we change the version or tell people to change the path themselves.)

Reply:
I have added the "generic installation instruction" first, and then use 1.0 as **example**, so I guess this should be ok?

Reply:
Thanks!